src/video.tsx

```diff
@@ -220,8 +218,6 @@ const Video = React.forwardRef<VideoElement, VideoProps>(
                 // This transparent pixel fallback avoids showing the ugly "play" image in android webviews
                 poster={TRANSPARENT_PIXEL}
                 style={{
-                    // For some reason adding this style with classnames doesn't add the border radius in safari
```

Comment:
I tried to replicate this, but it's working well when setting the border radius through classname for me in iOS and in safari desktop using a virtual machine. I've used [this playroom example](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8BlALgQzAawAQGcAHLGAXmAEYA2AXwD4AdAO11yQCEIAPGKAJQgB3Zq1ZoAlmgA2ZBiAAqkmXJGjY%2BMACdxhCRCak5AERgbtu8fpUtR%2BgAqbT%2BcgAoAlLlJ1cwGjVWsGPj4MGjkAaJIAGrisBARogSaYIYgABZoaITwAPQ5kAC2BfpQGJj4aBCaGADmMAB0NRAQNTIYhOL49YU5NWgAbgC0-bEwEPiDAEYArjihOfgYBYQyOfIwGJr4APIAZugwMFL1ywAs1omJaTDiNRnkABwADP42l7hBhDBgaHxllqlqPAAJwXS45RhvXCvVgQ1Qcbi4YhQKDiJg1AAyMF2YUotEhiWio3iUNY%2BGSqQyWVy%2BQgRRKZSClWqdUazVaGw6XR6fSGIziExmczQCyWKxgaw2Wz2ByOJ0I5xACVw11u92Azxhl0%2B31%2B-wggKoILBsIJbBynC4kKQOXQWGwkJAABoQGhrgVTAgANogACyEBGFU2zpAMX6EBDYYgAH0AHIwQQgAC6LsEsTd%2BG9AGYAExPJM0IA)

![Screenshot from 2024-02-15 14-18-54](https://github.com/Telefonica/mistica-web/assets/25785151/db581fbc-c3fe-4196-8f90-a0d912f3250a)

Reply:
:crossed_fingers: